sklearn/metrics/cluster/bicluster.py

```diff
@@ -1,7 +1,7 @@
 
 import numpy as np
 
-from sklearn.utils.linear_assignment_ import linear_assignment
+from scipy.optimize import linear_sum_assignment
 from sklearn.utils.validation import check_consistent_length, check_array
```

Comment:
It's not you, but can you please change this toa  relative import:

```from ...utils.validation import check_consistent_length, check_array```

Also Nitpick for pep8 compliance: the scipy import should be right below the numpy import, and there should be a blank line before the sklearn import:

```py
import numpy as np
from scipy.optimize import linear_sum_assignment

from ...utils.validation import check_consistent_length, check_array
```

Reply:
done